pkg/services/store/testdata/public_testdata.golden.jsonc

```diff
@@ -12,17 +12,17 @@
 //  }
 //  Name: 
 //  Dimensions: 3 Fields by 3 Rows
-//  +----------------------------+----------------------+---------------+
```

Comment:
you need to revert these changes, sometimes this gets automatically modified but the CI complains about it.